lms/static/sass/bootstrap/_variables.scss

```diff
@@ -13,3 +13,32 @@ $baseline: 20px !default;
 $header-logo-width: 60px !default;
 $header-logo-height: 40px !default;
 $header-user-image-size: 40px !default;
+
+// ----------------------------
+// #COLORS- Bootstrap-style
+// ----------------------------
+$state-success-text: $black !default;
```

Comment:
@amalbas I suppose that if you are adding all this new definitions, you are actually getting them from some other place. We should be removing them there, but I don't see this in this PR.

Could you please link the place where this was before?